source/arche/tutorial/testing.rst

```diff
@@ -73,18 +73,11 @@ You can then test the test's initial condition
     EXPECT_DOUBLE_EQ(facility->input.quantity(), qty);
     EXPECT_DOUBLE_EQ(facility->inventory.quantity(), 0);
 
-Now your test should look like
+Then execute the Tock:
 
 .. code-block:: c++
 
-    TEST_F(StorageTest, Tock) {
-      double qty = 42;
-      facility->input.Push(cyclus::NewBlankMaterial(qty));
-      EXPECT_DOUBLE_EQ(facility->input.quantity(), qty);
-      EXPECT_DOUBLE_EQ(facility->inventory.quantity(), 0);
       EXPECT_NO_THROW(facility->Tock());
-      // Test Storage specific behaviors of the Tock function here
-    }
 
```

Comment:
I had to replace all "facility" with "src_facility_" in the test for the Tock function or else the build failed. Was it working for  you with "facility" @mbmcgarry ?